src/d3a/d3a_core/sim_results/area_throughput_stats.py

```diff
@@ -68,9 +72,11 @@ def update_results(self, area_dict, core_stats, current_market_time_slot_str):
             area_results["export"].update(
                 {'capacity_kWh': round_floats_for_ui(export_capacity)}
             )
+        area_throughput_time = {}
```

Comment:
Minor, but i was confused by the naming (thought that this is a time dict).
How about: `area_throughput_profile` or just `area_throughput`